aws/dms/cron-contacts-rollup-hierarchy-pii.json

```diff
@@ -1,25 +0,0 @@
-{
-  "rules": [
-    {
-      "rule-type": "transformation",
-      "rule-id": "1",
-      "rule-name": "1",
-      "rule-action": "rename",
-      "rule-target": "schema",
-      "object-locator": {
-        "schema-name": "pegasus"
-      },
-      "value": "pegasus_pii"
-    },
-    {
-      "rule-type": "selection",
-      "rule-id": "2",
-      "rule-name": "2",
-      "object-locator": {
-        "schema-name": "pegasus",
-        "table-name": "contact_rollups"
-      },
-      "rule-action": "include"
-    }
-  ]
-}
```

Comment:
`cron-contacts-rollup-hierarchy-pii` and `cron-user-progress-hierarchy` tasks already no longer existed in the account (they were previously made obsolete by other tasks).